### PR TITLE
Update pm2docs.json

### PR DESCRIPTION
**Pull request by robin-monnier**
# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### CODE REVIEW

File: configs/pm2docs.json [hunk 1]
```diff
@@ -47,14 +47,14 @@
     "text": "#main p, #main li"
   },
   "selectors_exclude": [
-    "#next-step",
-    "#next-step + p",
+    "#next-steps",
+    "#next-steps  p",
```
Comment (s-pace) [inline configs/pm2docs.json]: Don't you miss the ~ here?

File: configs/pm2docs.json [hunk 2]
```diff
@@ -47,14 +47,14 @@
...
     "#questions",
-    "#questions + p",
+    "#questions ~ p",
```
Comment (s-pace) [inline configs/pm2docs.json]: Warning, every p after #questions will be skipped. Not only the last one

**Comment by robin-monnier** [comment]
Thanks I had indeed forgotten the '~' !

File: configs/pm2docs.json (hunk 1 above)
Comment (robin-monnier) [reply configs/pm2docs.json]: Yes I had

**Pull request closed without merging**